### PR TITLE
Fixed back button on smaller screens in the report page #285

### PR DESCRIPTION
**Pull request by Yondiame**
# Description

Resolved the issue with the back button on the report page not being responsive on mobile devices for some dimensions 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: src/common/Back/Back.tsx [hunk 1]
```diff
@@ -15,9 +16,15 @@ function Back(): JSX.Element {
   };
 
   return (
-    <Button onClick={handleBack} style={{ height: 36 }} inverted={darkMode} type="button">
-      <Icon name="arrow left" />
-      {t('back')}
+    <Button
+      className={styles.button}
+      onClick={handleBack}
+      // style={{ height: 36 }}
```
Comment (diballesteros) [inline src/common/Back/Back.tsx:22]: Could you remove the empty comment?

**Review by diballesteros — CHANGES_REQUESTED** [review]
Looks great! Could you also remove the package-lock.json from the PR, we use the yarn.lock so it isn't necessary to have both

File: src/common/Back/Back.tsx (hunk 1 above)
Comment (Yondiame) [reply src/common/Back/Back.tsx]: These have been completed thanks

**Review by diballesteros — APPROVED** [review]
Thanks!

**Pull request merged**